qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/expression/Parts.java

```diff
@@ -78,10 +78,42 @@ public NamespacePart getNamespacePart() {
 		}
 		return null;
 	}
-	
+
+	/**
+	 * Returns the property part of the parts and null otherwise.
+	 *
+	 * @return the property part of the parts and null otherwise.
+	 */
+	public PropertyPart getPropertyPart() {
+		if (super.getChildCount() == 0) {

```

Comment:
Please remove this method which is wrong because you can have several property part.